CMakeLists.txt

```diff
@@ -40,6 +40,9 @@ if (PYTHON STREQUAL "PYTHON-NOTFOUND")
     message(FATAL_ERROR "PYTHON not found: Please install Python 2.7.9 or later from https://www.python.org/downloads/")
 endif()
 
+# Track CMake CoreCLR targets in global property
+set_property(GLOBAL PROPERTY CLR_CMAKE_CORECLR_TARGETS "")
```

Comment:
With your latest change, you don't use the property for anything - you just fill it in. So I am a bit confused about this change.

Reply:
I have removed everything as per @weshaggard suggestion. The property `CLR_CMAKE_CORECLR_TARGETS` is left for future use in creating optimal build graph. If it should be removed as well and added later than I will do it.

There is still an open problem of reverting usage of `_add_xxx` wrappers. The question is in my previous comment at point 2: https://github.com/dotnet/coreclr/pull/14578#issuecomment-341141762

Reply:
@4creators 
> This may be very useful in optimization of project build graph.

I am not sure how we would do that. The build orchestration is fully in the hands of cmake. The difference in the build time between Windows and Linux is likely due to the difference between the performance of the compilers and the underlying build tools (make vs msbuild). 

So I think that we should remove the added property, the related changes in the _add_xxx functions and also the _add_custom_target function. As for the new usages of the _add_library and _add_executable wrappers that you've added and the related additions to the CLR_CROSS_COMPONENTS_LIST, I would keep these.

Reply:
OK Will do it